nilearn/conftest.py

```diff
@@ -7,21 +7,16 @@
 import pandas as pd
 import pytest
 from nibabel import Nifti1Image
+from scipy.signal import get_window
 
 from nilearn import image
-from nilearn._utils.data_gen import (
-    generate_fake_fmri,
-    generate_labeled_regions,
-    generate_maps,
-)
 from nilearn._utils.helpers import is_matplotlib_installed
 
 # we need to import these fixtures even if not used in this module
 from nilearn.datasets.tests._testing import (
     request_mocker,  # noqa: F401
```

Comment:
moving the request mocker should be moved too but it's a much bigger change so I want to do it in a separate PR